docker/run_tests_in_docker.sh

```diff
@@ -107,6 +107,6 @@ docker run \
        --tty \
        --interactive \
        --entrypoint= \
-       "${TESTIMAGE}" sh -c /app/docker/run_tests.sh
+       "${TESTIMAGE}" /app/docker/run_tests.sh
```

Comment:
This additional `sh -c` caused all the configman environment variables to get dropped because they aren't valid shell variables.